docker/README.md

```diff
@@ -11,46 +11,53 @@
   - [11: Scale by adding containers](#11-scale-by-adding-containers)
   - [12: Log to STDOUT or an external agent](#12-log-to-stdout-or-an-external-agent)
 
->**When to use Docker**
+> **When to use Docker**
 >
->Any developer is welcome to use Docker in their development setup. New apps developed at Abtion should not require to have a docker setup unless there is a good reason for it (like an uncommon dependency).
->Our own legacy apps might require a docker setup, however, we will strive to not make it a requirement if possible.
-
+> Any developer is welcome to use Docker in their development setup. New apps developed at Abtion should not require to have a docker setup unless there is a good reason for it (like an uncommon dependency).
+> Our own legacy apps might require a docker setup, however, we will strive to not make it a requirement if possible.
 
 # Dockerizing
 
+EVIL MARTIANS has done a lot of great progress making docker setup for rails. The examples files in this folder has originated from them. Other relevant links would be:
+
+https://github.com/evilmartians/fullstaq-ruby-docker
+
+https://github.com/bibendi/dip
+
```

Comment:
I would leave this part out.
I don't see the resemblance between fullstaq-ruby-docker and the example files in this diff.
What is the added value of using dip? I have never seen it before.